packages/playground/src/App.js

```diff
@@ -117,6 +110,32 @@ export default class extends React.Component {
     const sendBox = current && current.querySelector('input[type="text"]');
 
     sendBox && sendBox.focus();
+
+    const speech = new URLSearchParams(window.location.search).get('speech');
+
+    if (speech === 'bingspeech') {
+      const fetchAuthorizationToken = memoize(() => {
```

Comment:
Fetch new authorization token only after 1 minute. The token is valid for 10 minutes, however.

Using memoizer + custom equality function to cache the result makes the code simple and straightforward.